### ISSUE TITLE
No meeting Feb 18th, action items instead

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/bootstrap#104]
Hey All,

So tomorrow is a national holiday in both Canada and the USA, as we appeared to have consensus on the majority of topics necessary to draft the bylaws I think we can skip the meeting.

The foundations have been working on a draft that will be available very soon, in the meantime I've gone ahead and updated all the Stage-1 proposals to match what is going to be drafted and opened Pull Requests to move them to stage 2. Please chime in on those PRs to add your +1, or if anything seems out of place request changes.

The Node.js Board has a meeting on Friday where I hope we can get pre-approval to move all of these proposals to Stage-3. I believe the JSF has a meeting of sorts coming up soon as well, but I have also heard they are able to review this stuff out of band.

So assuming everyone gets their comments in before Friday, we should be able to land all of them as Stage-3 next Monday 🎉.

Community Board Reps to stage 2: https://github.com/nodejs/bootstrap/pull/103
CPC Charter to stage 2: https://github.com/nodejs/bootstrap/pull/100
Individual Membership to stage 2: https://github.com/nodejs/bootstrap/pull/102

TL;DR (lmk if I missed anything important)

**Community Board Reps**

* 1 board seat for CPC
* 1 floating board seat
  - process for seat determined annually by CPC
  - requires 2 Platinum board members to activate
* 1 board seat for individual membership program
  - requires 2 platinum board members and 2000 program members to activate

**CPC**

* CPC is made up of
  - Voting members (2 members per Impact Stage project, 2 for non-Impact projects, and 2 nominated by
     regular members)
  - Regular members (any member of any foundation project an get added to list of regular members)
  - Observers (any member of any foundation project)
* All membership types can attend any meeting and participate in consensus process
* If there is a vote only voting members participate
* CPC has a chair and director (director attends board meetings)
* CPC has a second board seat that is allocated on an annual basis
* CPC sets foundation wide policy
* CPC is final authority to decide if projects join foundation
* CPC does not have control of technical direction of projects within the foundation
* CPC oversees budget for foundation projects
* CPC will charter a team to help with CoC escalations within foundation.

**Individual Membership**

* CPC will be chartered by board with administering the program (in bylaws)
* Program will have board seat that activates when:
  - there are 3 platinum directors
  - there are 2000 members registered in the program
* Program will initially have one membership type but will expand to 3 if program is successful

**Comment by mhdawson** [comment]
I think the last thing that needs to be aligned between the docs is:

In CPC charter:
```
There are 3 Board seats earmarked for representation from the
_$FOUNDATION_ projects.  The first of these board seats is filled by
the CPC Director as outlined in Section 7 on [Elections][].

The second Board seat will be filled based on processes defined
by the CPC and approved by the Voting CPC members and then the Board. The
CPC members will review and either re-confirm or update the
processes once per year.

For the first year after the formation of the _$FOUNDATION_, the
processes for filling the remaining 1 Board seats will be as follows:

* As a constellation member of the _$FOUNDATION_, the Node.js project
  will elect one Board representative from either the TSC or
  Community Committee. The representative will be selected through a vote
  of the members of the Node.js organization. 

* One board member will be elected from the voting CPC members representing
  the Impact projects (excluding those representing Node.js) within
  the _$FOUNDATION_ through a vote of the members of the Impact
  projects (excluding Node.js).

This approach ensures that votes are not swamped by the larger nubmers of
members within the Node.js project while at the same time recognizing the
size and contribution of the Node.js project at this time.
```

and this in the board representation  - https://github.com/MylesBorins/bootstrap/tree/CPC-charter-stage-2/proposals/stage-1/COMMUNITY_BOARD_REPRESENTATION
```
There would be up to 3 community board seats:

    1 board seat for CPC
    1 floating board seat to be delegated by CPC
        delegated on an annual basis and approved by Board
    1 board seat for the individual membership program
```

**Comment by mhdawson** [comment]
Talking to Myles sounds like that was discussed in the last meeting.  I'll add a commit to Myle's PR to move to stage 2 in order to reflect what is in the COMMUNITY_BOARD_REPRESENTATION doc.
